Steepfile

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+D = Steep::Diagnostic
+
+target :lib do
+  signature "sig"
+
+  check "lib"
+
+  ignore "lib/prism/serialize.rb"
```

Comment:
Does this file need to be ignored? If it's just for making progress and we want to come back to it later that's fine, let's just notate it with a TODO.

Reply:
I've ignored additional files than the previous rev and added a TODO comment (but I've kept the WIP private RBS files for them).

This gets us down to 0 issues during `rake typecheck:steep`. I'll work on these files in future PRs if that's okay.